Directory.Packages.props

```diff
@@ -0,0 +1,46 @@
+<Project>
+  <ItemGroup>
+    <PackageVersion Include="FSharp.Core" Version="5.0.2" />
+    <PackageVersion Include="Microsoft.CodeCoverage" Version="16.11" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="16.11" />
+    <PackageVersion Include="Microsoft.NETCore.Platforms" Version="5.0.2" />
+    <PackageVersion Include="Microsoft.NETCore.Targets" Version="5.0" />
+    <PackageVersion Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" />
+    <PackageVersion Include="Microsoft.TestPlatform.ObjectModel" Version="16.11" />
+    <PackageVersion Include="MSTest.TestAdapter" Version="2.2.7" />
+    <PackageVersion Include="MSTest.TestFramework" Version="2.2.7" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageVersion Include="NuGet.Frameworks" Version="5.11" />
+    <PackageVersion Include="NUnit" Version="[3.13.1,4.0.0)" />
+    <PackageVersion Include="NUnit3TestAdapter" Version="4.0" />
+    <PackageVersion Include="System.Buffers" Version="4.5.1" />
+    <PackageVersion Include="System.Collections" Version="4.3" />
+    <PackageVersion Include="System.Collections.Immutable" Version="5.0" />
+    <PackageVersion Include="System.Diagnostics.Debug" Version="4.3" />
+    <PackageVersion Include="System.Diagnostics.TextWriterTraceListener" Version="4.3" />
+    <PackageVersion Include="System.Diagnostics.TraceSource" Version="4.3" />
+    <PackageVersion Include="System.Globalization" Version="4.3" />
+    <PackageVersion Include="System.IO" Version="4.3" />
+    <PackageVersion Include="System.Memory" Version="4.5.4" />
+    <PackageVersion Include="System.Numerics.Vectors" Version="4.5" />
+    <PackageVersion Include="System.Reflection" Version="4.3" />
+    <PackageVersion Include="System.Reflection.Metadata" Version="5.0" />
+    <PackageVersion Include="System.Reflection.Primitives" Version="4.3" />
+    <PackageVersion Include="System.Resources.ResourceManager" Version="4.3" />
+    <PackageVersion Include="System.Runtime" Version="4.3.1" />
+    <PackageVersion Include="System.Runtime.CompilerServices.Unsafe" Version="5.0" />
+    <PackageVersion Include="System.Runtime.Extensions" Version="4.3.1" />
+    <PackageVersion Include="System.Text.Encoding" Version="4.3" />
+    <PackageVersion Include="System.Threading" Version="4.3" />
+    <PackageVersion Include="System.Threading.Tasks" Version="4.3" />
+    <PackageVersion Include="Unquote" Version="6.0" />
+    <PackageVersion Include="xunit" Version="2.4.1" />
+    <PackageVersion Include="xunit.abstractions" Version="2.0.3" />
+    <PackageVersion Include="xunit.analyzers" Version="0.10" />
+    <PackageVersion Include="xunit.assert" Version="2.4.1" />
+    <PackageVersion Include="xunit.core" Version="2.4.1" />
+    <PackageVersion Include="xunit.extensibility.core" Version="2.4.1" />
+    <PackageVersion Include="xunit.extensibility.execution" Version="[2.4.1, 3.0.0)" />
```

Comment:
This is not a straight lift from paket.lock. It should match the dependencies of FsCheck.Xunit on master (but #645 means that's not the same as the dependencies in prod).